config/services.yaml

```diff
@@ -0,0 +1,54 @@
+# This file is the entry point to configure your own services.
+# Files in the packages/ subdirectory configure your dependencies.
+
+# Put parameters here that don't need to change on each machine where the app is deployed
+# https://symfony.com/doc/current/best_practices/configuration.html#application-related-configuration
+parameters:
+    symfony_docs_secret: '%env(SYMFONY_DOCS_SECRET)%'
+
+imports:
+    - { resource: github.yml }
+
+services:
+    # default configuration for services in *this* file
+    _defaults:
+        autowire: true      # Automatically injects dependencies in your services.
+        autoconfigure: true # Automatically registers your services as commands, event subscribers, etc.
+
+    # makes classes in src/ available to be used as services
+    # this creates a service per class whose id is the fully-qualified class name
+    App\:
+        resource: '../src/*'
+        exclude: '../src/{DependencyInjection,Subscriber,Issues,Entity,Migrations,Event,Tests,Kernel.php,GitHubEvents.php}'
```

Comment:
No reason to over-optimize the exclude:

* I think Subscriber can be removed if we remove `github.yml` and allow those to be auto-registered like normal
* I'd remove `Issues`. Sure, some things are model classes, but it doesn't hurt to include them. Plus, then it would auto-register `GitHubRequestHandler` and the 2 services in the `Github/` sub-directory

Reply:
Later